### ISSUE TITLE
How to render aligned math equations?

### ISSUE THREAD

**Issue by AntixK** [miyuchina/mistletoe#218]
Hi,

I am trying to render aligned math equations from markdown to HTML. For example

```
This is a markdown file.

$$
\begin{align}
f &= ma \\
e &= mc^2
\end{align}
$$
```
Specifically, the main issue is that the new line in the math block is tokenized as `</br>` in the output html and Katex fails to render. 

Thanks for creating and maintaining this great library!

**Comment by pbodnar** [comment]
@AntixK, can you share code snippet / are you sure you are using the [MathJaxRenderer](https://github.com/miyuchina/mistletoe/blob/master/mistletoe/contrib/mathjax.py#L9)? For me, it outputs the expected output:

```html
<p>$$
\begin{align}
f &amp;= ma \\
e &amp;= mc^2
\end{align}
$$</p>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-MML-AM_CHTML"></script>
```

**Comment by AntixK** [comment]
Hi, thanks, that works. I was trying out a custom rendered for math with labels. I missed out the mathjax renderer.
